README.md

```diff
@@ -15,8 +15,8 @@ the square brackets (for multiple-choice questions).
 
 What does it mean for an HTML tag to be "semantic"? Give an example of a short snippet of HTML written using semantic tags and non-semantic tags.
 
-```text
-Your answer here.
+```
+Tags that can be understood by a browser while still allowing the developer to also understand what is going on within the code.  For a non-semantic I want to say div or ul or I think a p tag as well.  While a semantic one is like table or header tags like h1-h6.
 ```
```

Comment:
I really would like you to expand on this further. Specifically being "Semantic" is describing the function of the content rather than the style of it. I think a stronger example might be using `<strong>`which is semantic over `<b>` which is non-semantic. Strong is describing importance of the html content, while b is simply used to differentiate the styles of the content